File: Data Structures & Algorithms/is-subsequence/submission-0.cpp
class Solution {
public:
    bool isSubsequence(string s, string t) {
        int idx = 0;
        int res = s.size();
        for(int i = 0 ; i < t.size() ; i++){
            if(t[i] == s[idx]){
                idx++;
                res--;
            }
        }
        return (res == 0);
    }
};

File: Data Structures & Algorithms/longest-palindromic-substring/submission-0.py
class Solution:
    def longestPalindrome(self, s: str) -> str:
        res = ""
        resLen = 0
        n = len(s)

        #odd length
        for i in range(n):
            r,l = i,i
            while r < n and l >= 0 and s[l] == s[r]:
                if r-l+1 > resLen:
                    res = s[l:r+1]
                    resLen = r-l+1
                r+=1
                l-=1
        
        #even length
        for i in range(n):
            r,l = i+1,i
            while r < n and l >= 0 and s[l] == s[r]:
                if r-l+1 > resLen:
                    res = s[l:r+1]
                    resLen = r-l+1
                r+=1
                l-=1

        return res